tests/installation/setup_zdup.pm

```diff
@@ -4,7 +4,13 @@ use testapi;
 
 sub run() {
     # wait booted
-    assert_screen 'generic-desktop', 200;
+    if (get_var("NOAUTOLOGIN")) {
+        mouse_hide();
+        assert_screen 'displaymanager', 200;
```

Comment:
the mouse_hide is pretty useless as this will be called when you still have the bootloader displayed.